app/assets/javascripts/application.js

```diff
@@ -75,5 +75,15 @@ angular.module("OpenHq", ['ngRoute', 'ngAnimate', 'ngSanitize', 'rails', 'ngFile
 
   // Don’t convert attributes to camel case
   railsSerializerProvider.underscore(angular.identity).camelize(angular.identity);
+})
 
+.run(function($rootScope, $mdDialog) {
```

Comment:
Theres maybe a nicer place to put this, but since its pretty much a global function, i figured this would be the best place for now?